web/viewer.js

```diff
@@ -996,7 +980,7 @@ var PDFViewerApplication = {
     if (this.pdfViewer.currentScale === UNKNOWN_SCALE) {
```

Comment:
Since this patch changes `PDFViewer.currentScale` so that it never returns `UNKNOWN_SCALE` any more, this line will no longer work. This should explain the zooming issue, on initial load, that @timvandermeij mentioned.

Should we perhaps instead use `if (!this.pdfViewer.currentScaleValue) {` here, since according to [pdf_viewer.js#L327](https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.js#L327) that will be `null` until the scale has been set, or is another solution preferred?